init.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import zipfile
+import io
+import os
+import sys
+
+repos_url = "https://api.github.com/users/"
+repo_zip_url = "https://github.com/"
+
+def getRepos(username=None):
+    print ("Getting repository list")
+    try:
+        url = repos_url + username + "/repos"
+        response = requests.get(url)
+        repos = response.json()
+        return repos
+    except Exception as e:
+        print (e)
+   
+def saveRepos(username, repos=None):
+    if repos:
+        abspath = os.path.abspath(sys.path[0])
```

Comment:
```python
sys.path[0] = "/usr/local/lib/python2.7/dist-packages/git/ext/gitdb"
```
Not sure if you meant to use sys.path[1] which is the folder path to the python script.